setup.py

```diff
@@ -35,29 +35,38 @@
     from distutils.command.build_ext import build_ext
 
 
+
+# Use a provided libarchive else default to hard-coded path.
+libarchivePrefix = environ.get('LIBARCHIVE_PREFIX')
+
+if libarchivePrefix:
+    includePath = libarchivePrefix + '/include'
+else:
+    includePath = '/usr/local/include'
+
+
+
+
 name = 'python-libarchive'
 version = '4.0.1'
-release = '1'
+release = '2'

```

Comment:
Can this be set back to 1? I'll change the version when doing the release to PyPI

Reply:
Done